AudioLink/Shaders/AudioLinkTestTime.shader

```diff
@@ -2,6 +2,7 @@
 {
     Properties
     {
+        _AudioTexture("_AudioTexture", 2D) = "white" {}
```

Comment:
What is this for? Are you passing in the rendertexture explicitly? Not sure how properties will interact with existing named grabpasses

Reply:
if _AudioTexture is defined as a Property it will be sampled instead of the named grabpass
this is only viable for non-UGC, by just assigning the texture, or if you set the property via script

Reply:
@yewnyx do not commit this change.